_sass/imports/_general.scss

```diff
@@ -1,11 +1,11 @@
 html {
   font-family: $font-primary;
-  font-size: 100%;
+  font-size: 18px;

```

Comment:
So good accessibility question here... if a user has set a different default font size for their browser (because they like bigger text overall as an example) and we are setting an absolute pixel size that overrides their decision for our design system is that a good or bad thing?
By retaining our starting font-size as a percentage (112.5%) at least everything is sized relative to not just each other but also what the user has set as their default?

Reply:
Excellent point, using percentages is probably the more accessibility-friendly option. Added a new commit.